cmd/nerdctl/run.go

```diff
@@ -966,6 +966,35 @@ func parseKVStringsMapFromLogOpt(cmd *cobra.Command, logDriver string) (map[stri
 		if _, ok := logOptMap[logging.MaxSize]; !ok {
 			delete(logOptMap, logging.MaxFile)
 		}
+		for _, value := range logging.FluentdBitLogOpts {
+			if _, ok := logOptMap[value]; ok {
```

Comment:
May be it is easier to make the opposite way  by checking the number of drivers and if more than 1 we log that only the first driver is considered

Reply:
emmm I'm not get the point of `checking the number of drivers `

Reply:
plz take a look https://github.com/containerd/nerdctl/pull/1244/files#r922888056